test/battle/ability/sheer_force.c

```diff
@@ -8,7 +8,8 @@ SINGLE_BATTLE_TEST("Sheer Force boosts power, but removes secondary effects of m
 
     for (j = 1; j < MOVES_COUNT; j++)
     {
-        if (gBattleMoves[j].sheerForceBoost && j != MOVE_ORDER_UP)
+        if (gBattleMoves[j].sheerForceBoost && j != MOVE_ORDER_UP && j != MOVE_AURA_WHEEL
+          && gBattleMoves[j].effect != EFFECT_PLACEHOLDER)
```

Comment:
Why aren't Order Up or Aura Wheel not counted for this?
```suggestion
        if (gBattleMoves[j].sheerForceBoost
          //&& gBattleMoves[j].effect != EFFECT_ORDER_UP
          && gBattleMoves[j].effect != EFFECT_AURA_WHEEL
          && gBattleMoves[j].effect != EFFECT_PLACEHOLDER)
```

Reply:
I think Order Up was added before I added the EFFECT_PLACEHOLDER check. MOVE_AURA_WHEEL isn't an option because it fails if the user isn't Morpeko.